cmd/titus-storage/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"log"
+	"os"
+)
+
+func main() {
+	command := os.Args[1]
```

Comment:
You should use `flags.Parse` or cobra.

Reply:
Ok. Just seemed like a lot of complexity for 1 cli argument.
I don't want to use flags because it doesn't support bare args.

Reply:
I'm wondering if you might want to, since there are a bunch of `os.GetEnv()`s in various places in this code that you could consolidate into the cobra parsing.  You could then log the total config for the CLI tool (based on flags and Env vars) in one place, which would be nice for debugging.

Reply:
That is true, cobra + viper does have some magic to turn variables into cli constructs.

I'll look to see how hard it is to put these together to remove the os.getEnvs()